docs/pages/versions/unversioned/sdk/age-range.mdx

```diff
@@ -0,0 +1,81 @@
+---
+title: AgeRange
+description: A library that provides access to age range information using Google Play Age Signals API on Android and Declared Age Range framework on iOS.
+sourceCodeUrl: 'https://github.com/expo/expo/tree/main/packages/expo-age-range'
+packageName: 'expo-age-range'
+platforms: ['android', 'ios']
+isAlpha: true
+---
+
+import APISection from '~/components/plugins/APISection';
+import { APIInstallSection } from '~/components/plugins/InstallSection';
+import { ConfigReactNative } from '~/ui/components/ConfigSection';
+import { SnackInline } from '~/ui/components/Snippet';
+
+`expo-age-range` provides access to user age range information. It uses Google's [Play Age Signals API](https://developer.android.com/google/play/age-signals/use-age-signals-api) on Android and Apple's [Declared Age Range framework](https://developer.apple.com/documentation/declaredagerange/) on iOS.
+
+This API allows you to request age range information from users to help you comply with age-appropriate content regulations and provide age-appropriate experiences in your app.
+
+Limitation: On Android, the Play Age Signals API (beta) throws an exception until January 1, 2026, [as documented](https://developer.android.com/google/play/age-signals/use-age-signals-api#integrate-play-age-signals). From January 1, the API will return live responses.
+
+## Installation
+
+<APIInstallSection />
+
+## Configuration in app config
+
+### Setup iOS project
+
+To use the age range API on iOS, you need to add the `com.apple.developer.declared-age-range` entitlement to your app. Add it to your **app.json** or **app.config.js**:
+
+```json app.json
+{
+  "expo": {
+    "ios": {
+      "entitlements": {
+        "com.apple.developer.declared-age-range": true
+      }
+    }
+  }
+}
+```
+
+<ConfigReactNative>
+
+For bare React Native projects, add the entitlement to your **ios/[app]/[app].entitlements** file:
+
+```xml
+<key>com.apple.developer.declared-age-range</key>
+<true/>
+```
+
+</ConfigReactNative>
+
+## Usage
+
+<SnackInline label="Age Range Usage" dependencies={['expo-age-range']} platforms={['android', 'ios', 'web']}>
```

Comment:
```suggestion
<SnackInline label="Age Range Usage" dependencies={['expo-age-range']} platforms={['android', 'ios']}>
```

- Since the web is an unsupported platform as per the platform tags, we should remove the platform tag here.
- The example below opens in Snack (not sure if that is intentional). However, the example is incomplete to run inside a Snack (missing primitives to render the functionality and the UI).

Reply:
A Snack example will be great here, thank you @vonovak!

If we can provide runnable examples of libraries that work in Expo Go, that is always a win-win for developers to get familiar with the library 😄